httpclient.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"context"
+	"net/http"
+)
+
+// Inspired by: https://www.thegreatcodeadventure.com/mocking-http-requests-in-golang/
+
+type httpClient interface {
+	Do(req *http.Request) (*http.Response, error)
+}
+
+var (
+	client httpClient = &http.Client{}
+)
+
+// Get an HTTP response by sending an HTTP GET request to the specified URL.
+func Get(url string, headers http.Header) (*http.Response, error) {
```

Comment:
I would add this to environment to keep all the logic in there, maybe a `doGet` function?

Reply:
Good idea. Added.